js/piwik.js

```diff
@@ -7084,7 +7084,10 @@ if (typeof window.Piwik !== 'object') {
                             apply(paq[iterator]);
                             delete paq[iterator];
 
-                            if (appliedMethods[methodName] > 1) {
+                            var isMethodCallableMultipleTimes = ["addTracker"],
+                                shouldMethodBeCalledOnceOnly = isMethodCallableMultipleTimes.indexOf(methodName) === -1;
```

Comment:
you want to use `indexOfArray` as .indexOf is not available in all browsers. 
Also be good to use to `var` statements. `isMethodCallableMultipleTimes ` be better renamed to `methodsCallableMultipleTimes` as it is not a boolean.

In theory it would be even doable to just say `methodName !='addTracker'` as more is not needed right now.